content/livesync/connector/database.textile

```diff
@@ -0,0 +1,198 @@
+---
+title: Database setup
+meta_description: "The Ably Database Connector requires two tables in your database to run. See how to create each table and the privileges required to utilize them."
+product: livesync
+---
+
+The Ably Database Connector requires two tables to exist in your database, an "outbox table":#outbox and a "nodes table":#nodes.
+
+h2(#create). Create tables
+
+The required tables can be created using the "CLI":#cli or created "automatically":#auto on startup of the Database Connector.
+
+h3(#cli). Using the CLI
+
+Use the @CREATE TABLE@ command to automatically create the required tables in your database. This requires the application to use a PostgreSQL user that has been granted @CREATE@ permissions on the desired schemas.
```

Comment:
Capitalised `CREATE TABLE` refers to the underlying SQL statement, but here we are using the CLI command `create tables`

Reply:
Good catch, I've updated it back to lowercase plural for the CLI command!